tsconfig.json

```diff
@@ -9,24 +9,16 @@
     "declaration": true,
     "noImplicitAny": true,
     "removeComments": false,
-    "noLib": true,
     "preserveConstEnums": true,
     "suppressImplicitAnyIndexErrors": false,
     "outDir": "lib",
     "rootDir": "src"
   },
-  "filesGlob": [
-    "node_modules/typescript/lib/lib.core.es7.d.ts",
-    "custom_typings/*.d.ts",
-    "src/**/*.ts",
-    "typings/index.d.ts"
-  ],
   "files": [
```

Comment:
🎉 nice! It's behavior is a little weird, which is why I hadn't been able to get it working before.  Files still need to be referenced in the `files` array.